src/plugins/workspace/public/components/workspace_creator/workspace_creator_form.tsx

```diff
@@ -138,28 +147,47 @@ export const WorkspaceCreatorForm = (props: WorkspaceCreatorFormProps) => {
             </>
           )}
           {permissionEnabled && (
-            <>
-              <EuiTitle
-                {...generateRightSidebarScrollProps(RightSidebarScrollField.Member)}
-                size="s"
-              >
-                <h3>{usersAndPermissionsCreatePageTitle}</h3>
-              </EuiTitle>
-              <EuiText size="xs">
-                {i18n.translate('workspace.creator.form.usersAndPermissionsDescription', {
-                  defaultMessage:
-                    'You will be added as an owner to the workspace. Select additional users and user groups as workspace collaborators with different access levels.',
-                })}
-              </EuiText>
-              <EuiSpacer size="m" />
-              <WorkspacePermissionSettingPanel
-                errors={formErrors.permissionSettings?.fields}
-                onChange={setPermissionSettings}
-                permissionSettings={formData.permissionSettings}
-                disabledUserOrGroupInputIds={disabledUserOrGroupInputIdsRef.current}
-                data-test-subj={`workspaceForm-permissionSettingPanel`}
-              />
-            </>
+            <EuiSplitPanel.Outer direction="column">
```

Comment:
Any reason to use SplitPanel? Can we use `EuiPanel` instead?

Reply:
No particular reason, I just happened to see that SplitPanel can achieve this effect. I'll change it to EuiPanel then.

Reply:
It seems there will be a 32px gap between `SplitPanel.Inner`. Let use `EuiPanel` with `EuiSpacer` instead.

Reply:
ok